JHipster fails to deploy to Heroku

##### **Overview of the issue**

I tried my blog demo this morning and when I try to deploy it to Heroku, the following error occurs:

```
2017-09-21T14:36:07.876258+00:00 app[web.1]: Setting JAVA_TOOL_OPTIONS defaults based on dyno size. Custom settings will override them.
2017-09-21T14:36:07.884303+00:00 app[web.1]: Picked up JAVA_TOOL_OPTIONS: -Xmx300m -Xss512k -Dfile.encoding=UTF-8
2017-09-21T14:36:16.700011+00:00 app[web.1]:
2017-09-21T14:36:16.700026+00:00 app[web.1]:         ██╗ ██╗   ██╗ ████████╗ ███████╗   ██████╗ ████████╗ ████████╗ ███████╗
2017-09-21T14:36:16.700027+00:00 app[web.1]:         ██║ ██║   ██║ ╚══██╔══╝ ██╔═══██╗ ██╔════╝ ╚══██╔══╝ ██╔═════╝ ██╔═══██╗
2017-09-21T14:36:16.700034+00:00 app[web.1]:         ██║ ████████║    ██║    ███████╔╝ ╚█████╗     ██║    ██████╗   ███████╔╝
2017-09-21T14:36:16.700035+00:00 app[web.1]:   ██╗   ██║ ██╔═══██║    ██║    ██╔════╝   ╚═══██╗    ██║    ██╔═══╝   ██╔══██║
2017-09-21T14:36:16.700036+00:00 app[web.1]:   ╚██████╔╝ ██║   ██║ ████████╗ ██║       ██████╔╝    ██║    ████████╗ ██║  ╚██╗
2017-09-21T14:36:16.700036+00:00 app[web.1]:    ╚═════╝  ╚═╝   ╚═╝ ╚═══════╝ ╚═╝       ╚═════╝     ╚═╝    ╚═══════╝ ╚═╝   ╚═╝
2017-09-21T14:36:16.700037+00:00 app[web.1]:
2017-09-21T14:36:16.700038+00:00 app[web.1]: :: JHipster 🤓  :: Running Spring Boot 1.5.7.RELEASE ::
2017-09-21T14:36:16.700040+00:00 app[web.1]: :: http://www.jhipster.tech ::
2017-09-21T14:36:16.700802+00:00 app[web.1]:
2017-09-21T14:36:18.346146+00:00 app[web.1]: 2017-09-21 14:36:18.340  INFO 4 --- [           main] com.okta.developer.BlogApp               : Starting BlogApp v0.0.1-SNAPSHOT on 9035a629-f1b0-498c-9a82-249ebc10a8d1 with PID 4 (/app/target/blog-0.0.1-SNAPSHOT.war started by u44816 in /app)
2017-09-21T14:36:18.346801+00:00 app[web.1]: 2017-09-21 14:36:18.346  INFO 4 --- [           main] com.okta.developer.BlogApp               : The following profiles are active: prod,heroku
2017-09-21T14:36:49.644168+00:00 app[web.1]: 2017-09-21 14:36:49.630  WARN 4 --- [           main] ationConfigEmbeddedWebApplicationContext : Exception encountered during context initialization - cancelling refresh attempt: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'com.ryantenney.metrics.spring.config.annotation.DelegatingMetricsConfiguration': Unsatisfied dependency expressed through method 'setMetricsConfigurers' parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'metricsConfiguration': Unsatisfied dependency expressed through method 'setHikariDataSource' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is java.lang.IllegalArgumentException: URL must start with 'jdbc'
2017-09-21T14:36:49.821866+00:00 app[web.1]: 2017-09-21 14:36:49.813 ERROR 4 --- [           main] o.s.b.f.s.DefaultListableBeanFactory     : Destroy method on bean with name 'org.springframework.context.annotation.internalScheduledAnnotationProcessor' threw an exception
2017-09-21T14:36:49.821869+00:00 app[web.1]:
2017-09-21T14:36:49.821870+00:00 app[web.1]: java.lang.IllegalStateException: ApplicationEventMulticaster not initialized - call 'refresh' before multicasting events via the context: org.springframework.boot.context.embedded.AnnotationConfigEmbeddedWebApplicationContext@44f75083: startup date [Thu Sep 21 14:36:19 UTC 2017]; root of context hierarchy
2017-09-21T14:36:49.821871+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.getApplicationEventMulticaster(AbstractApplicationContext.java:414)
2017-09-21T14:36:49.821872+00:00 app[web.1]: 	at org.springframework.context.support.ApplicationListenerDetector.postProcessBeforeDestruction(ApplicationListenerDetector.java:97)
2017-09-21T14:36:49.821873+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:253)
2017-09-21T14:36:49.821874+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:578)
2017-09-21T14:36:49.821875+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:554)
2017-09-21T14:36:49.821875+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:961)
2017-09-21T14:36:49.821876+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:523)
2017-09-21T14:36:49.821877+00:00 app[web.1]: 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.destroySingletons(FactoryBeanRegistrySupport.java:230)
2017-09-21T14:36:49.821877+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:968)
2017-09-21T14:36:49.821878+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1030)
2017-09-21T14:36:49.821879+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:556)
2017-09-21T14:36:49.821879+00:00 app[web.1]: 	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122)
2017-09-21T14:36:49.821880+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:693)
2017-09-21T14:36:49.821881+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:360)
2017-09-21T14:36:49.821882+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:303)
2017-09-21T14:36:49.821883+00:00 app[web.1]: 	at com.okta.developer.BlogApp.main(BlogApp.java:66)
2017-09-21T14:36:49.821883+00:00 app[web.1]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2017-09-21T14:36:49.821884+00:00 app[web.1]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2017-09-21T14:36:49.821885+00:00 app[web.1]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2017-09-21T14:36:49.821885+00:00 app[web.1]: 	at java.lang.reflect.Method.invoke(Method.java:498)
2017-09-21T14:36:49.821886+00:00 app[web.1]: 	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
2017-09-21T14:36:49.821887+00:00 app[web.1]: 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
2017-09-21T14:36:49.821887+00:00 app[web.1]: 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
2017-09-21T14:36:49.821888+00:00 app[web.1]: 	at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59)
2017-09-21T14:36:49.821889+00:00 app[web.1]:
2017-09-21T14:36:49.832391+00:00 app[web.1]: 2017-09-21 14:36:49.828 ERROR 4 --- [           main] o.s.b.f.s.DefaultListableBeanFactory     : Destroy method on bean with name 'org.springframework.boot.context.properties.ConfigurationPropertiesBindingPostProcessor' threw an exception
2017-09-21T14:36:49.832394+00:00 app[web.1]:
2017-09-21T14:36:49.832396+00:00 app[web.1]: java.lang.IllegalStateException: ApplicationEventMulticaster not initialized - call 'refresh' before multicasting events via the context: org.springframework.boot.context.embedded.AnnotationConfigEmbeddedWebApplicationContext@44f75083: startup date [Thu Sep 21 14:36:19 UTC 2017]; root of context hierarchy
2017-09-21T14:36:49.832397+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.getApplicationEventMulticaster(AbstractApplicationContext.java:414)
2017-09-21T14:36:49.832398+00:00 app[web.1]: 	at org.springframework.context.support.ApplicationListenerDetector.postProcessBeforeDestruction(ApplicationListenerDetector.java:97)
2017-09-21T14:36:49.832399+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:253)
2017-09-21T14:36:49.832400+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:578)
2017-09-21T14:36:49.832401+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:554)
2017-09-21T14:36:49.832402+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:961)
2017-09-21T14:36:49.832403+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:523)
2017-09-21T14:36:49.832404+00:00 app[web.1]: 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.destroySingletons(FactoryBeanRegistrySupport.java:230)
2017-09-21T14:36:49.832404+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:968)
2017-09-21T14:36:49.832405+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1030)
2017-09-21T14:36:49.832406+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:556)
2017-09-21T14:36:49.832406+00:00 app[web.1]: 	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122)
2017-09-21T14:36:49.832407+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:693)
2017-09-21T14:36:49.832408+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:360)
2017-09-21T14:36:49.832412+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:303)
2017-09-21T14:36:49.832413+00:00 app[web.1]: 	at com.okta.developer.BlogApp.main(BlogApp.java:66)
2017-09-21T14:36:49.832413+00:00 app[web.1]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2017-09-21T14:36:49.832414+00:00 app[web.1]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2017-09-21T14:36:49.832415+00:00 app[web.1]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2017-09-21T14:36:49.832415+00:00 app[web.1]: 	at java.lang.reflect.Method.invoke(Method.java:498)
2017-09-21T14:36:49.832416+00:00 app[web.1]: 	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
2017-09-21T14:36:49.832417+00:00 app[web.1]: 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
2017-09-21T14:36:49.832417+00:00 app[web.1]: 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
2017-09-21T14:36:49.832418+00:00 app[web.1]: 	at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59)
2017-09-21T14:36:49.832418+00:00 app[web.1]:
2017-09-21T14:36:49.842049+00:00 app[web.1]: 2017-09-21 14:36:49.841 ERROR 4 --- [           main] o.s.b.f.s.DefaultListableBeanFactory     : Destroy method on bean with name 'org.springframework.boot.autoconfigure.internalCachingMetadataReaderFactory' threw an exception
2017-09-21T14:36:49.842052+00:00 app[web.1]:
2017-09-21T14:36:49.842054+00:00 app[web.1]: java.lang.IllegalStateException: ApplicationEventMulticaster not initialized - call 'refresh' before multicasting events via the context: org.springframework.boot.context.embedded.AnnotationConfigEmbeddedWebApplicationContext@44f75083: startup date [Thu Sep 21 14:36:19 UTC 2017]; root of context hierarchy
2017-09-21T14:36:49.842055+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.getApplicationEventMulticaster(AbstractApplicationContext.java:414)
2017-09-21T14:36:49.842055+00:00 app[web.1]: 	at org.springframework.context.support.ApplicationListenerDetector.postProcessBeforeDestruction(ApplicationListenerDetector.java:97)
2017-09-21T14:36:49.842056+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DisposableBeanAdapter.destroy(DisposableBeanAdapter.java:253)
2017-09-21T14:36:49.842057+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroyBean(DefaultSingletonBeanRegistry.java:578)
2017-09-21T14:36:49.842058+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingleton(DefaultSingletonBeanRegistry.java:554)
2017-09-21T14:36:49.842059+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingleton(DefaultListableBeanFactory.java:961)
2017-09-21T14:36:49.842060+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.destroySingletons(DefaultSingletonBeanRegistry.java:523)
2017-09-21T14:36:49.842060+00:00 app[web.1]: 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.destroySingletons(FactoryBeanRegistrySupport.java:230)
2017-09-21T14:36:49.842061+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.destroySingletons(DefaultListableBeanFactory.java:968)
2017-09-21T14:36:49.842062+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.destroyBeans(AbstractApplicationContext.java:1030)
2017-09-21T14:36:49.842062+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:556)
2017-09-21T14:36:49.842063+00:00 app[web.1]: 	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122)
2017-09-21T14:36:49.842064+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:693)
2017-09-21T14:36:49.842065+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:360)
2017-09-21T14:36:49.842066+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:303)
2017-09-21T14:36:49.842066+00:00 app[web.1]: 	at com.okta.developer.BlogApp.main(BlogApp.java:66)
2017-09-21T14:36:49.842067+00:00 app[web.1]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2017-09-21T14:36:49.842068+00:00 app[web.1]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2017-09-21T14:36:49.842068+00:00 app[web.1]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2017-09-21T14:36:49.842069+00:00 app[web.1]: 	at java.lang.reflect.Method.invoke(Method.java:498)
2017-09-21T14:36:49.842070+00:00 app[web.1]: 	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
2017-09-21T14:36:49.842071+00:00 app[web.1]: 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
2017-09-21T14:36:49.842071+00:00 app[web.1]: 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
2017-09-21T14:36:49.842072+00:00 app[web.1]: 	at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59)
2017-09-21T14:36:49.842073+00:00 app[web.1]:
2017-09-21T14:36:50.150083+00:00 app[web.1]: 2017-09-21 14:36:50.147 ERROR 4 --- [           main] o.s.boot.SpringApplication               : Application startup failed
2017-09-21T14:36:50.150094+00:00 app[web.1]:
2017-09-21T14:36:50.150097+00:00 app[web.1]: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'com.ryantenney.metrics.spring.config.annotation.DelegatingMetricsConfiguration': Unsatisfied dependency expressed through method 'setMetricsConfigurers' parameter 0; nested exception is org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'metricsConfiguration': Unsatisfied dependency expressed through method 'setHikariDataSource' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is java.lang.IllegalArgumentException: URL must start with 'jdbc'
2017-09-21T14:36:50.150098+00:00 app[web.1]: 	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.inject(AutowiredAnnotationBeanPostProcessor.java:667)
2017-09-21T14:36:50.150099+00:00 app[web.1]: 	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88)
2017-09-21T14:36:50.150100+00:00 app[web.1]: 	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:366)
2017-09-21T14:36:50.150101+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1264)
2017-09-21T14:36:50.150101+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
2017-09-21T14:36:50.150102+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
2017-09-21T14:36:50.150103+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
2017-09-21T14:36:50.150103+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
2017-09-21T14:36:50.150104+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
2017-09-21T14:36:50.150105+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
2017-09-21T14:36:50.150105+00:00 app[web.1]: 	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:372)
2017-09-21T14:36:50.150106+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1173)
2017-09-21T14:36:50.150107+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1067)
2017-09-21T14:36:50.150107+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:513)
2017-09-21T14:36:50.150108+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
2017-09-21T14:36:50.150109+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
2017-09-21T14:36:50.150110+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
2017-09-21T14:36:50.150111+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
2017-09-21T14:36:50.150111+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
2017-09-21T14:36:50.150112+00:00 app[web.1]: 	at org.springframework.context.support.PostProcessorRegistrationDelegate.registerBeanPostProcessors(PostProcessorRegistrationDelegate.java:235)
2017-09-21T14:36:50.150113+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.registerBeanPostProcessors(AbstractApplicationContext.java:703)
2017-09-21T14:36:50.150113+00:00 app[web.1]: 	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:528)
2017-09-21T14:36:50.150119+00:00 app[web.1]: 	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122)
2017-09-21T14:36:50.150120+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:693)
2017-09-21T14:36:50.150121+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:360)
2017-09-21T14:36:50.150122+00:00 app[web.1]: 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:303)
2017-09-21T14:36:50.150122+00:00 app[web.1]: 	at com.okta.developer.BlogApp.main(BlogApp.java:66)
2017-09-21T14:36:50.150123+00:00 app[web.1]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2017-09-21T14:36:50.150124+00:00 app[web.1]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2017-09-21T14:36:50.150124+00:00 app[web.1]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2017-09-21T14:36:50.150125+00:00 app[web.1]: 	at java.lang.reflect.Method.invoke(Method.java:498)
2017-09-21T14:36:50.150126+00:00 app[web.1]: 	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48)
2017-09-21T14:36:50.150126+00:00 app[web.1]: 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87)
2017-09-21T14:36:50.150127+00:00 app[web.1]: 	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50)
2017-09-21T14:36:50.150128+00:00 app[web.1]: 	at org.springframework.boot.loader.WarLauncher.main(WarLauncher.java:59)
2017-09-21T14:36:50.150129+00:00 app[web.1]: Caused by: org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'metricsConfiguration': Unsatisfied dependency expressed through method 'setHikariDataSource' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is java.lang.IllegalArgumentException: URL must start with 'jdbc'
2017-09-21T14:36:50.150130+00:00 app[web.1]: 	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.inject(AutowiredAnnotationBeanPostProcessor.java:667)
2017-09-21T14:36:50.150131+00:00 app[web.1]: 	at org.springframework.beans.factory.annotation.InjectionMetadata.inject(InjectionMetadata.java:88)
2017-09-21T14:36:50.150131+00:00 app[web.1]: 	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor.postProcessPropertyValues(AutowiredAnnotationBeanPostProcessor.java:366)
2017-09-21T14:36:50.150132+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1264)
2017-09-21T14:36:50.150133+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:553)
2017-09-21T14:36:50.150134+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
2017-09-21T14:36:50.150134+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
2017-09-21T14:36:50.150135+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
2017-09-21T14:36:50.150136+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
2017-09-21T14:36:50.150137+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
2017-09-21T14:36:50.150137+00:00 app[web.1]: 	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:208)
2017-09-21T14:36:50.150138+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.addCandidateEntry(DefaultListableBeanFactory.java:1316)
2017-09-21T14:36:50.150139+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.findAutowireCandidates(DefaultListableBeanFactory.java:1282)
2017-09-21T14:36:50.150139+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveMultipleBeans(DefaultListableBeanFactory.java:1180)
2017-09-21T14:36:50.150140+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1096)
2017-09-21T14:36:50.150141+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1066)
2017-09-21T14:36:50.150141+00:00 app[web.1]: 	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.inject(AutowiredAnnotationBeanPostProcessor.java:659)
2017-09-21T14:36:50.150142+00:00 app[web.1]: 	... 34 common frames omitted
2017-09-21T14:36:50.150146+00:00 app[web.1]: Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'dataSource' defined in class path resource [org/springframework/boot/autoconfigure/jdbc/DataSourceConfiguration$Hikari.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is java.lang.IllegalArgumentException: URL must start with 'jdbc'
2017-09-21T14:36:50.150147+00:00 app[web.1]: 	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:599)
2017-09-21T14:36:50.150148+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.instantiateUsingFactoryMethod(AbstractAutowireCapableBeanFactory.java:1173)
2017-09-21T14:36:50.150150+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBeanInstance(AbstractAutowireCapableBeanFactory.java:1067)
2017-09-21T14:36:50.150151+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:513)
2017-09-21T14:36:50.150152+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:483)
2017-09-21T14:36:50.150152+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:306)
2017-09-21T14:36:50.150153+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
2017-09-21T14:36:50.150154+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:302)
2017-09-21T14:36:50.150154+00:00 app[web.1]: 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
2017-09-21T14:36:50.150155+00:00 app[web.1]: 	at org.springframework.beans.factory.config.DependencyDescriptor.resolveCandidate(DependencyDescriptor.java:208)
2017-09-21T14:36:50.150156+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.doResolveDependency(DefaultListableBeanFactory.java:1138)
2017-09-21T14:36:50.150156+00:00 app[web.1]: 	at org.springframework.beans.factory.support.DefaultListableBeanFactory.resolveDependency(DefaultListableBeanFactory.java:1066)
2017-09-21T14:36:50.150157+00:00 app[web.1]: 	at org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor$AutowiredMethodElement.inject(AutowiredAnnotationBeanPostProcessor.java:659)
2017-09-21T14:36:50.150158+00:00 app[web.1]: 	... 50 common frames omitted
2017-09-21T14:36:50.150159+00:00 app[web.1]: Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.zaxxer.hikari.HikariDataSource]: Factory method 'dataSource' threw exception; nested exception is java.lang.IllegalArgumentException: URL must start with 'jdbc'
2017-09-21T14:36:50.150160+00:00 app[web.1]: 	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:189)
2017-09-21T14:36:50.150160+00:00 app[web.1]: 	at org.springframework.beans.factory.support.ConstructorResolver.instantiateUsingFactoryMethod(ConstructorResolver.java:588)
2017-09-21T14:36:50.150161+00:00 app[web.1]: 	... 62 common frames omitted
2017-09-21T14:36:50.150162+00:00 app[web.1]: Caused by: java.lang.IllegalArgumentException: URL must start with 'jdbc'
2017-09-21T14:36:50.150162+00:00 app[web.1]: 	at org.springframework.util.Assert.isTrue(Assert.java:92)
2017-09-21T14:36:50.150163+00:00 app[web.1]: 	at org.springframework.boot.jdbc.DatabaseDriver.fromJdbcUrl(DatabaseDriver.java:268)
2017-09-21T14:36:50.150164+00:00 app[web.1]: 	at org.springframework.boot.autoconfigure.jdbc.DataSourceProperties.determineDriverClassName(DataSourceProperties.java:238)
2017-09-21T14:36:50.150165+00:00 app[web.1]: 	at org.springframework.boot.autoconfigure.jdbc.DataSourceProperties.initializeDataSourceBuilder(DataSourceProperties.java:183)
2017-09-21T14:36:50.150165+00:00 app[web.1]: 	at org.springframework.boot.autoconfigure.jdbc.DataSourceConfiguration.createDataSource(DataSourceConfiguration.java:42)
2017-09-21T14:36:50.150166+00:00 app[web.1]: 	at org.springframework.boot.autoconfigure.jdbc.DataSourceConfiguration$Hikari.dataSource(DataSourceConfiguration.java:80)
2017-09-21T14:36:50.150167+00:00 app[web.1]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2017-09-21T14:36:50.150167+00:00 app[web.1]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2017-09-21T14:36:50.150168+00:00 app[web.1]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2017-09-21T14:36:50.150176+00:00 app[web.1]: 	at java.lang.reflect.Method.invoke(Method.java:498)
2017-09-21T14:36:50.150177+00:00 app[web.1]: 	at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:162)
2017-09-21T14:36:50.150177+00:00 app[web.1]: 	... 63 common frames omitted
2017-09-21T14:36:50.150178+00:00 app[web.1]:
2017-09-21T14:36:50.795902+00:00 heroku[web.1]: Process exited with status 1
2017-09-21T14:36:50.811164+00:00 heroku[web.1]: State changed from starting to crashed
```

##### **Motivation for or Use Case**

It used to work. @jkutner Any ideas?

##### **Reproduce the error**

```
git clone https://github.com/mraible/jhipster-heroku
yo jhipster:heroku
```

##### **JHipster Version(s)**

4.8.2
Yes, it is a regression.

##### **JHipster Version(s)**

```
blog@0.0.0 /Users/mraible/blog
└── generator-jhipster@4.8.2

```


##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**


<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.okta.developer",
      "nativeLanguage": "en"
    },
    "jhipsterVersion": "4.8.2",
    "baseName": "blog",
    "packageName": "com.okta.developer",
    "packageFolder": "com/okta/developer",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "hibernateCache": "ehcache",
    "clusteredHttpSession": false,
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "mysql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSocialSignIn": false,
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "replaced-by-jhipster-info",
    "clientFramework": "angularX",
    "useSass": false,
    "clientPackageManager": "yarn",
    "applicationType": "monolith",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": [
      "en"
    ]
  }
}
</pre>
</details>


##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>
entity Blog (blog) {
  name String required minlength(3),
  handle String required minlength(2)
}
entity Entry (entry) {
  title String required,
  content byte[] required,
  date ZonedDateTime required
}
entity Tag (tag) {
  name String required minlength(2)
}

relationship ManyToOne {
  Blog{user(login)} to User,
  Entry{blog(name)} to Blog
}
relationship ManyToMany {
  Entry{tag(name)} to Tag{entry}
}

paginate Entry with infinite-scroll
paginate Tag with infinite-scroll

</pre>
</details>


##### **Environment and Tools**

openjdk version "1.8.0_144"
OpenJDK Runtime Environment (Zulu 8.23.0.3-macosx) (build 1.8.0_144-b01)
OpenJDK 64-Bit Server VM (Zulu 8.23.0.3-macosx) (build 25.144-b01, mixed mode)

git version 2.14.1

node: v8.4.0

npm: 5.4.1

gulp:
[10:42:13] CLI version 1.4.0

yeoman: 2.0.0

yarn: 0.24.4

Docker version 17.06.2-ce, build cec0b72

docker-compose version 1.14.0, build c7bdf9e

Execution complete

- [x] Checking this box is mandatory (this is just to show you read everything)

--- a/generators/heroku/index.js
+++ b/generators/heroku/index.js
@@ -248,9 +248,9 @@ module.exports = HerokuGenerator.extend({
             if (this.prodDatabaseType === 'postgresql') {
                 dbAddOn = 'heroku-postgresql --as DATABASE';
             } else if (this.prodDatabaseType === 'mysql') {
-                dbAddOn = 'jawsdb:kitefin --as JAWSDB';
+                dbAddOn = 'jawsdb:kitefin --as DATABASE';
             } else if (this.prodDatabaseType === 'mariadb') {
-                dbAddOn = 'jawsdb-maria:kitefin --as JAWSDB';
+                dbAddOn = 'jawsdb-maria:kitefin --as DATABASE';
             } else if (this.prodDatabaseType === 'mongodb') {
                 dbAddOn = 'mongolab:sandbox --as MONGODB';
             } else {
@@ -258,7 +258,7 @@ module.exports = HerokuGenerator.extend({
             }
 
             this.log(chalk.bold('\nProvisioning addons'));
-            exec(`heroku addons:create ${dbAddOn} --as DATABASE --app ${this.herokuAppName}`, {}, (err, stdout, stderr) => {
+            exec(`heroku addons:create ${dbAddOn} --app ${this.herokuAppName}`, {}, (err, stdout, stderr) => {
                 if (err) {
                     const verifyAccountUrl = 'https://heroku.com/verify';
                     if (_.includes(err, verifyAccountUrl)) {

